.pre-commit-config.yaml

```diff
@@ -64,7 +64,7 @@ repos:
         name: Typecheck
         always_run: true
         entry: yarn --cwd frontend tsc --noEmit
-        files: \.js$|\.jsx$|\.ts$|\.tsx$
+        files: \.(js$|jsx$|ts$|tsx)$
```

Comment:
nit: I think the `$` at the end of the pattern should make those inside of it unnecessary

Reply:
OMG, right!